ui/app/components/app/permission-page-container/permission-page-container-content/permission-page-container-content.component.js

```diff
@@ -12,63 +12,69 @@ export default class PermissionPageContainerContent extends PureComponent {
     selectedPermissions: PropTypes.object.isRequired,
     permissionsDescriptions: PropTypes.object.isRequired,
     onPermissionToggle: PropTypes.func.isRequired,
-    selectedAccount: PropTypes.object,
+    selectedIdentities: PropTypes.array,
     redirect: PropTypes.bool,
     permissionRejected: PropTypes.bool,
   }
 
   static defaultProps = {
     redirect: null,
     permissionRejected: null,
-    selectedAccount: {},
+    selectedIdentities: [],
   }
 
   static contextTypes = {
     t: PropTypes.func,
   }
 
-  renderAccountInfo = (account) => {
+  renderRedirectVisual () {
+    const { permissionRejected, selectedIdentities, domainMetadata } = this.props
     return (
-      <div className="permission-approval-visual__account-info">
-        <div className="permission-approval-visual__account-info__label">
-          { account.label }
+      <div className="permission-approval-visual__redirect">
+        <IconWithFallBack icon={domainMetadata.icon} name={domainMetadata.name} />
+        <div className="permission-approval-visual__redirect-center">
+          { permissionRejected
+            ? <span className="permission-approval-visual__reject" ><i className="fa fa-times-circle" /></span>
+            : <span className="permission-approval-visual__check" />
+          }
+          <img className="permission-approval-visual__broken-line" src="/images/broken-line.svg" />
```

Comment:
This line seems to overlap with the site icon on the left

Reply:
Screenshot:
![line-overlap](https://user-images.githubusercontent.com/2459287/78056093-e84f8d80-735a-11ea-939c-0f5b4c8e0c29.png)

Reply:
I believe this has been fixed

![Screenshot from 2020-04-01 04-01-39](https://user-images.githubusercontent.com/7499938/78106207-82501e00-73cd-11ea-87e0-3591bc5c8cbc.png)